src/test/multiAuth.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+
+import * as auth from "../auth";
+import { configstore } from "../configstore";
+
+describe("multiAuth", () => {
```

Comment:
The name of this file (and this block) are confusing me. The `.spec` files should match source files in the repo. Could this match something better? `auth.spec.ts`?